catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/alias/AttributeAliases.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.alias;
+
+import java.util.Map;
+
+/**
+ * A collection of mappings between internal attribute names to their user-friendly aliases,
+ * typically used for display in the UI.
+ *
+ * <p><b> This code is experimental. While this interface is functional and tested, it may change or
+ * be removed in a future version of the library. </b>
+ */
+public interface AttributeAliases {
+
+  /**
+   * Checks if the internal attribute {@code attributeName} has a configured alias.
+   *
+   * @param attributeName - the internal name of the attribute
+   * @return true if {@code attributeName} has a configured alias, false otherwise.
+   */
+  boolean hasAlias(String attributeName);
```

Comment:
This seems redundant, since the return value of `getAlias` can also be used to test for the presence of an alias as well.

Reply:
It's true it's redundant, but I think the added readability is worth it (i.e. `if(hasAlias(attr))` vs `if(getAlias(attr) != null)`)